.github/workflows/build.yaml

```diff
@@ -0,0 +1,24 @@
+name: Build ARM user
+
+on: [pull_request, workflow_dispatch]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Install deps
+      run: |
+        apt --no-install-recommends -y build-dep qemu
+        apt install autoconf libtool protobuf-c-compiler
+        opam install piqi
+        cs /root
```

Comment:
perhaps you mean `cd`? I would suggest staying at home, as otherwise you will have a ton of issues as you're not run as a super user here.